internal/events/eventer.go

```diff
@@ -61,6 +61,13 @@ const (
 	metricsSubsystem = "eventer"
 )
 
+const (
+	// ExecuteEntityEventTopic is the topic for internal webhook events
+	ExecuteEntityEventTopic = "execute.entity.event"
```

Comment:
I wasn't sure about moving these constants here, but I couldn't find another place that would be outside the top-level engine package and wouldn't cause yet another import loop

Reply:
This works for now. We could move it to a `constants` package if we see the need for it.